avm/res/machine-learning-services/workspace/tests/e2e/ai/main.test.bicep

```diff
@@ -65,26 +65,29 @@ module testDeployment '../../../main.bicep' = [
       associatedStorageAccountResourceId: nestedDependencies.outputs.storageAccountResourceId
       sku: 'Basic'
       kind: 'Hub'
-      connections: [
-        {
-          name: 'ai'
-          category: 'AIServices'
-          target: nestedDependencies.outputs.aiServicesEndpoint
-          connectionProperties: {
-            authType: 'ApiKey'
-            credentials: {
-              key: 'key'
-            }
-          }
-          metadata: {
-            ApiType: 'Azure'
-            ResourceId: nestedDependencies.outputs.aiServicesResourceId
-            Location: enforcedLocation
-            ApiVersion: '2023-07-01-preview'
-            DeploymentApiVersion: '2023-10-01-preview'
-          }
-        }
-      ]
+      // There currently appears to be a bug that fails the idempotent deployment if it runs
+      // immediately after the initial deployment with no delay, when the connection is defined.
+      // The connection is commented out for now, but should be uncommented once the bug is fixed.
```

Comment:
Did you, by any chance, open a bug anywhere that we could link here to find out if set bug was fixed?

Reply:
I haven't
I would have thought that for something like this you would have to raise a support ticket, but I didn't get round to it

Reply:
I've raised a support ticket